web-app/deployment/docker-compose.yaml

```diff
@@ -0,0 +1,38 @@
+version: "3"
```

Comment:
You can combine your docker-compose in one line command

It's very useful to create some shared config in docker compose file.

example:

`docker-compose -f docker-compose.yml -f docker-compose.dev.yml` will merge this tow files to make only one config.
It means that you can have a main docker-compose defining all of your services and both docker-compose.dev.yml or docker-compose.prod.yml just have to override some params that you need for eachs envs (port, volumes etc) Don't hesitate to call me if you need a real world example, I have one :)